src/scripts.js

```diff
@@ -84,7 +91,25 @@ function getDate() {
   return myDate;
 };
 
-function resetHydrationData() {
+function resetWidgetData() {
   hydrationDataLatestWeek.innerText = '';
   showHydrationData();
+  sleepLatestWeekData.innerText = '';
+  showSleepData();
 }
+
+function showSleepData() {
```

Comment:
let's consider splitting this `showSleepData()` into two functions if we have time, it's super long!